msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AbstractApplicationBase.java

```diff
@@ -319,6 +325,17 @@ public T readTimeoutForDefaultHttpClient(Integer val) {
             return self();
         }
 
+        /**
+         * Disables the library's internal retry logic for HTTP requests. Used alongside
+         *
+         * @param val timeout value in milliseconds
+         * @return instance of the Builder on which method was called
+         */
+        public T disableInternalRetries(boolean val) {
+            disableInternalRetries = val;
+            return self();
+        }
```

Comment:
I don't think this should accept a param? It's only function should be to disable internal retries, and that should happen once on the initial application configuration. I think this should just set disableInternalRetries to`true`.

Reply:
Yeah, seems like msal-node and .NET both have a configuration file separate from the builder, but for MSAL Java it's all in the same place.

(there's no technical/Java reason why we couldn't also split it up, it's just not how the public API was originally designed)